css/properties/display.json

```diff
@@ -208,6 +209,7 @@
           "__compat": {
             "description": "<code>&lt;display-outside&gt;</code>",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/display-outside",
+            "spec_url": "https://drafts.csswg.org/css-display/#typedef-display-outside",
```

Comment:
I included this by mistake, should revert, but doesn't indicate a problem with anything except my cherry picking skills.